packages/toolkit/src/config/environments.default.json

```diff
@@ -1,20 +1,17 @@
 {
```

Comment:
For the aragonCLI and Toolkit we used the concept of `environment` where we had configurations for each specific network that were mostly Aragon related. Given the current vision of having `console.aragon` we most likely want to stop using `environment`.